Add Nat's January Jams blog post (#791)
godotengine/godot-website: collections/_article/january-jams-2024.md

---
title: "January Jams"

> excerpt: "Looking back at a month of GameJams, we curated a list of ten events that you might or might not have heard of before."

Reviewer: ```suggestion
excerpt: "Looking back at a month of game jams, we curated a list of ten events that you might or might not have heard of before."
```
Again, a matter of subjective taste, but "GameJams" made it sound like that was a particular branded thing.

Reviewer 2: Same here, it looks better the way you suggested it. Thank you!

categories: ["events"]
author: Nathalie Galla
image: /storage/blog/covers/godot-jam.webp

> image_caption_description: "Jam base image by <a href='https://www.freepik.com/free-photo/jam-isolated-white_1198739.htm#query=jam&position=39&from_view=search&track=sph&uuid=449f0bd4-ec2a-4a1b-aef3-d0ad383022a7'>luis_molinero</a> on Freepik"

Reviewer: ```suggestion
image_caption_description: "Jam base image by <a href='https://www.freepik.com/free-photo/jam-isolated-white_1198739.htm'>luis_molinero</a> on Freepik"
```
Removed search and tracking parameters.

Author: This one is a bit weird, because I get that when I copy paste the image credit, but then when trying to access it from a url without the tracking I wasn't able to load the page. So it seems to be somewhat required? If you can access it without the tracking parameters let me know and we can remove it 👍

Reviewer: Ugh, you're right. I had initially clicked the long link, then I deleted the extra parameters and checked it still worked - which it did, but only in that same browser. If I try the shorter link in a fresh browser it doesn't work. I guess the site set a cookie or something on the first visit which allowed the second visit to work on the first browser.

So, yeah, it looks like the full URL is needed. Sorry for not testing more completely. I just have a hatred for messy URLs and thought I could fix it!

Author: Hate it too :( thanks for flagging it!

date: 2024-01-31 12:00:00
---

Did you partake in a game jam this month? To us here it seemed like the beginning of 2024 was already jam-packed (yes, that pun is intentional) with opportunities to hone your game-making skills, in variable time spans from just three hours to an entire month.

Below you can find a list of our ten personal favorites from this busy month (in chronological order):

### 1. Gaming Like It’s 1928

🔗 [https://itch.io/jam/gaming-like-its-1928](https://itch.io/jam/gaming-like-its-1928)

> 📅 Janurary 1st - February 1st

Reviewer: ```suggestion
📅 January 1st - February 1st
```

Did you hear about Mickey Mouse entering the public domain this year? How is that relevant to this jam you ask, well… you can now include the character in your games without fear of repercussions. This is precisely what the organizers turned into a creative challenge, to use the newly freed IPs from 1928 (exactly 95 years ago, the current protection period) to make a digital or analog game.

### 2. Finally Finish Something (FFS) 2024

🔗 [https://itch.io/jam/finally-finish-something-2024](https://itch.io/jam/finally-finish-something-2024) 

📅 January 1st  - February 4th

> Be honest, how high is your stack of unfinished games that you once were sure would see the light of day? This particular game jam caters to your new year’s resolution to”Finally Finish Something” and hence allows you to submit any game you have previously been working on, as long as you have used the month of January to complete the project.

Reviewer: ```suggestion
Be honest, how high is your stack of unfinished games that you once were sure would see the light of day? This particular game jam caters to your new year’s resolution to ”Finally Finish Something” and hence allows you to submit any game you have previously been working on, as long as you have used the month of January to complete the project.
```

### 3. Queer Vampire Jam

🔗 [https://itch.io/jam/queer-vampire](https://itch.io/jam/queer-vampire) 

📅 January 10th  - February 20th

While there are a plethora of “niche” game jams at any given time, it seems important to highlight at least one focused on the LGBTQIA+ community. Also, vampires are cool, and this interactive-fiction jam is not only still ongoing, but also kind of popular. Plenty of time to still submit your own bloodsucker romance!

### 4. Game Dev Knock Out (GDKO) - Round 1

🔗 [https://itch.io/jam/gdko-2024](https://itch.io/jam/gdko-2024) 

📅 January 1st  - January 15th

> The Squid Games of the game dev world. The previous sentence is taken straight from their own branding, albeit it does seem nobody dies in this rendition and the price is a T-shirt. Every participant is sorted into one of five teams, spearheaded by the organizers themselves, which each adds a unique modifier to the common challenge.

Reviewer: ```suggestion
The Squid Games of the game dev world. The previous sentence is taken straight from their own branding, albeit it does seem nobody dies in this rendition and the prize is a T-shirt. Every participant is sorted into one of five teams, spearheaded by the organizers themselves, which each adds a unique modifier to the common challenge.
```
I assume it's a prize not an entry fee!

### 5. New Year, New Skills Game Jam

🔗 [https://itch.io/jam/new-year-new-skills-game-jam](https://itch.io/jam/new-year-new-skills-game-jam) 

📅 January 7th  - January 14th

An explicitly beginner-friendly jam organized by [Bitter Sails Games](https://twitter.com/bittersails) that quickly rose to the top of itch.io’s game jam suggestions. Introductory presentations and mentoring by game industry veterans allowed for a guided experience, while the resource package prepared by the team is so comprehensive we suggest downloading it even after the fact.

### 6. Godot Wild Jam #65

🔗 [https://itch.io/jam/godot-wild-jam-65](https://itch.io/jam/godot-wild-jam-65) 

📅 January 12th - January 21st

A staple in our community that takes place during the second week of every month. In addition to the announced theme, three wildcards are released to provide an optional additional challenge. Most importantly, you are required to use the Godot Engine for this one ;)

### 7. Pirate Software - Game Jam 14

🔗 [https://itch.io/jam/pirate](https://itch.io/jam/pirate)

📅 January 12th - January 27th

> A jam organized by Game Developer & Twitch Streamer [Pirate Software](https://twitter.com/PirateSoftware). Unique compared to the other events in this list is the requirement to provide a Game Design Document for your entry. On top of that, you can almost always find co-work companions in his live streams.

Reviewer: ```suggestion
A jam organized by game developer & Twitch streamer [Pirate Software](https://twitter.com/PirateSoftware). Unique compared to the other events in this list is the requirement to provide a game design document for your entry. On top of that, you can almost always find co-work companions in his live streams.
```
This is a matter of taste, but I think it looks better without so many capital letters.

Reviewer 2: Agreed - sometimes my German-ness comes through an makes me want to capitalize everything xD

From the numbers that are accessible to us, we can estimate that about __16% of submissions were made using the Godot Engine__. That amounts to over 1500 games, more than double compared to last year!

### 9. GGJ Next

🔗 [https://ggjnext.org/](https://ggjnext.org/) 

📅 January 22nd - January 28th

Concurrent to the Global Game Jam this event by the same organizers caters to the younger aspiring game developers aged 12 to 17. Furthermore, it provides a curriculum of video lessons about game development basics.

### 10. Trijam #254

🔗 [https://itch.io/jam/trijam-254](https://itch.io/jam/trijam-254) 

📅 January 26th - January 29th

Is it possible to make a game in just three hours? The hosts of this fast paced jam surely seem to think so. For those joining as a team, every member gets three hours to work on their part. Since this game jam requires the least time commitment out of our selection, and happens weekly, is there a good reason for you to not try and join the next one?

----

> Did you join any game jam(s) this month? Then don’t miss out on the opportunity to self-promote your projects under our __#JamJanuary__ posts which you can find here: [linkin.bio/godot](linkin.bio/godot).

Reviewer: ```suggestion
Did you join any game jams this month? Then don’t miss out on the opportunity to self-promote your projects under our __#JamJanuary__ posts which you can find here: [https://linkin.bio/godot](linkin.bio/godot).
```
I don't think the messy brackets are necessary when asking that question.
